diff --git a/dist/index.js b/dist/index.js
index 753b7b2a756791bc49769de0b462910c21907d21..7a053e9a1c86b8d4cb8546ff2ac086bb244e806d 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1722,7 +1722,7 @@ var SandboxApi = class {
           }
         },
         body: {
-          autoPause: false,
+          autoPause: opts.autoPause ?? false,
           timeout: this.timeoutToSeconds(timeoutMs)
         },
         signal: config.getSignal(opts == null ? void 0 : opts.requestTimeoutMs)
@@ -1746,7 +1746,7 @@ var SandboxApi = class {
       const client = new ApiClient(config);
       const res = yield client.api.POST("/sandboxes", {
         body: {
-          autoPause: false,
+          autoPause: opts.autoPause ?? false,
           templateID: template,
           metadata: opts == null ? void 0 : opts.metadata,
           envVars: opts == null ? void 0 : opts.envs,
diff --git a/dist/index.mjs b/dist/index.mjs
index 457ba46dd999aa4ce8ce9c1a42efbf45ab4f5301..9197f99c5668a7c6d83b88eea2e9351334e13de5 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1688,7 +1688,7 @@ var SandboxApi = class {
           }
         },
         body: {
-          autoPause: false,
+          autoPause: opts.autoPause ?? false,
           timeout: this.timeoutToSeconds(timeoutMs)
         },
         signal: config.getSignal(opts == null ? void 0 : opts.requestTimeoutMs)
@@ -1712,7 +1712,7 @@ var SandboxApi = class {
       const client = new ApiClient(config);
       const res = yield client.api.POST("/sandboxes", {
         body: {
-          autoPause: false,
+          autoPause: opts.autoPause ?? false,
           templateID: template,
           metadata: opts == null ? void 0 : opts.metadata,
           envVars: opts == null ? void 0 : opts.envs,
